configure.ac

```diff
@@ -1,22 +1,9 @@
+dnl SPDX-FileCopyrightText: 2013-2023 Reuben Thomas <rrt@sc3d.org>
```

Comment:
Please use `#` for comments rather than `dnl`.

Reply:
ok, will do